nncf/__init__.py

```diff
@@ -30,6 +30,16 @@
 except ImportError:
     tf = None
 
-if torch is None and tf is None:
+try:
+    import onnx
+except ImportError:
+    onnx = None
+
+try:
+    import openvino
```

Comment:
To ensure that openvino is properly installed, we should check `import openvino.runtime` and `import openvino.tools.pot` will not raise `ImportError`